conf/webknossos.latest.routes

```diff
@@ -100,7 +100,7 @@ PUT           /datastores/:name/datasource
 PUT           /datastores/:name/datasources                                             controllers.WKRemoteDataStoreController.updateAll(name: String, key: String)
 PATCH         /datastores/:name/status                                                  controllers.WKRemoteDataStoreController.statusUpdate(name: String, key: String)
 POST          /datastores/:name/reserveUpload                                           controllers.WKRemoteDataStoreController.reserveDataSetUpload(name: String, key: String, token: String)
-POST          /datastores/:name/reportDatasetUpload                                     controllers.WKRemoteDataStoreController.reportDatasetUpload(name: String, key: String, token: String, dataSetName: String, dataSetSizeBytes: Long)
+POST          /datastores/:name/reportDatasetUpload                                     controllers.WKRemoteDataStoreController.reportDatasetUpload(name: String, key: String, token: String, dataSetName: String, dataSetSizeBytes: Long, needsConversion: Boolean)
```

Comment:
Doesn't this need some front end change?

Reply:
Fortunately not, this route is only called by the datastore, not the frontend